tensorflow/python/eager/def_function.py

```diff
@@ -773,7 +773,35 @@ def _decorate(self, decorator):
     self._function_spec = function_lib.FunctionSpec.from_function_and_signature(
         self._python_function, self.input_signature)
 
-  def _get_tracing_count(self):
```

Comment:
It looks like there are a few user of this method, unfortunately (one is in `tensorflow/python/distribute/client/client.py`). There are two options:
 * try a global search for `_get_tracing_count` and update all uses 
 * keep both methods (and just have one call the other), along with a TODO to remove the private one